bool beacon_detected
int16 side
